Singer.API/Services/EventService.cs

```diff
@@ -34,5 +36,42 @@ public EventService(ApplicationDbContext context, IMapper mapper) : base(context
                f.Description.Contains(filter);
          return filterExpression;
       }
+
+      public IReadOnlyList<EventDescriptionDTO> GetPublicEvents(SearchEventParamsDTO searchEventParamsDto)
+      {
+         var today = DateTime.Today;
+
+         Expression<Func<Event, bool>> useStartDate =
+            x => x.StartDate >= today &&
+                 (!searchEventParamsDto.StartDate.HasValue || x.StartDate == searchEventParamsDto.StartDate.Value) &&
```

Comment:
2nd part should be `x.StartDate >= searchEventParamsDto.StartDate.Value`. Currently we're only showing events on the exact date you've chosen

Reply:
Well I choose that option deliberately: with having that implementation, you would see all events coming after that one. Not sure if that's desired.

Reply:
In that case, the UX is not clear enough :) The way it looked to me I assumed the StartDate field should be a 'from' and the EndDate field a 'to' filter

Reply:
Let's merge as-is and see how StG responds to it, perhaps it's just me misinterpreting the field

Reply:
No the UI is clear, because that's the point: you're interested in events starting on that date, not in the ones after that.
But this is a more functional requirement, so we'll ask the client :)